modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenConstants.java

```diff
@@ -205,12 +205,17 @@
     public static final String MODEL_PROPERTY_NAMING = "modelPropertyNaming";
     public static final String MODEL_PROPERTY_NAMING_DESC = "Naming convention for the property: 'camelCase', 'PascalCase', 'snake_case' and 'original', which keeps the original name";
 
+    public static final String PARAM_PROPERTY_NAMING = "paramPropertyNaming";
```

Comment:
the name sounds ambiguous - like it's naming for properties of parameters. but parameters do not have properties.
maybe just `param naming` instead?